nautobot/dcim/tests/test_models.py

```diff
@@ -696,6 +698,11 @@ def setUp(self):
         self.location_2 = Location.objects.create(
             name="Leaf", status=self.device_status, location_type=self.location_type_2, parent=self.location_1
         )
+        self.device_redundancy_group = DeviceRedundancyGroup.objects.create(
+            name="Redundancy Group 1",
+            status=Status.objects.get_for_model(DeviceRedundancyGroup).get(slug="active"),
+            failover_strategy=DeviceRedundancyGroupFailoverStrategyChoices.FAILOVER_ACTIVE_ACTIVE,
+        )
```

Comment:
```suggestion
        self.device_redundancy_group = DeviceRedundancyGroup.objects.first()
```

Since we have factory now.